app/state/reducers/ui/accessibilityReducer.js

```diff
@@ -17,6 +18,12 @@ function toggleContrast(state) {
   return { ...state, isHighContrast: !state.isHighContrast };
 }
 
+function setAccessibilityPreferences(state, preferences) {
+  const newPreferences = Array.isArray(preferences) ? preferences : [];
+  setSelA11yPref(newPreferences);
```

Comment:
I think should be pure functions and we should avoid performing side effects inside them [according to Redux docs](https://redux.js.org/basics/reducers#handling-actions).
Probably the subscriber should handle persisting and retrieving the state from the `localStorage`(https://stackoverflow.com/a/35675304/9441362)

Reply:
This is true, moved it to a separate middleware.